spec/requests/evenements_spec.rb

```diff
@@ -37,7 +37,7 @@
           .to change { Evenement.count }.by(1)
                                         .and change { Partie.count }.by(1)
         evenement = Evenement.last
-        expect(evenement.date).to eq DateTime.new(2019, 0o2, 25, 16, 11, 29)
+        expect(evenement.date.to_datetime).to eq Time.at(1_551_111_089, 238_000).to_datetime
```

Comment:
pour moi on ne tient pas compte des millisecondes ici. D'après la référence de l'internet il faudrait utiliser `strftime`
https://stackoverflow.com/questions/14107354/millisecond-resolution-of-datetime-in-ruby